src/writeout_nc.F

```diff
@@ -1643,15 +1643,17 @@ end subroutine writestat_nc
 !ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
 !ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
 
-      subroutine writedropdiag_nc(nrec,rtime,tnumpart,radavg,radsqr,radmin,radmax,Tpavg,Tpsqr,Tpmin,Tpmax,Tfavg,qfavg,prsavg,rhoavg,zhvec,numconc)
+      subroutine writedropdiag_nc(nrec,&
```

Comment:
@drichte2 I thought you would eventually remove all the netCDF interface in CM1?

Reply:
I'm still not sure what to do about this. We had discussed this a while ago, but I just kept going with the netCDF since I'm not sure how else to output the drop_diag stats